presentation/src/main/java/com/open/presentation/ui/components/CurrentWeatherDetailRow.kt

```diff
@@ -0,0 +1,49 @@
+package com.open.weather.ui.theme.ui.components
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Card
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun CurrentWeatherDetailRow(title1: String, value1: String, title2: String, value2: String) {
```

Comment:
Instead of using `1` and `2` in variable names, we can make them more descriptive by using `top`, `bottom`, or similar words. For example: `fun CurrentWeatherDetailRow(titleTop: String, valueTop: String, titleBottom: String, valueBottom: String)`